src/Common/src/TypeSystem/Ecma/EcmaModule.cs

```diff
@@ -24,6 +24,8 @@ EntityHandle Handle
             }
         }
 
+        internal ModuleDesc ReferencingModule { get; set; }
```

Comment:
It may be better to make this `EcmaAssembly ReferencingAssembly`. We expect this to always be an assembly.